frontend/src/components/Teams/MyTeams/partials/CardTeam/CardBody.tsx

```diff
@@ -34,15 +34,15 @@ const CardBody = React.memo<CardBodyProps>(({ userId, team }) => {
 	const { _id: id, users } = team;
 
 	const userIsParticipating = useMemo(() => {
-		return !!users.find((user) => user.user._id === userId);
+		return !!users.find((user) => user.user?._id === userId);
 	}, [users, userId]);
 
 	const havePermissions = useMemo(() => {
 		if (isSAdmin) {
 			return true;
 		}
 
-		const myUser = team.users.find((user) => String(user.user._id) === String(userId));
+		const myUser = team.users.find((user) => String(user.user?._id) === String(userId));
 
 		if (team && (myUser?.role === 'admin' || myUser?.role === 'stakeholder')) {
```

Comment:
As a suggestion I prefer to return the boolean like this:
```suggestion
		return team && (myUser?.role === 'admin' || myUser?.role === 'stakeholder');
```